Modules/Challenges/test/Challenges.API.Tests.Integration/Support/Dependencies.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Challenges.API.Tests.Integration.API;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using RestSharp;
+using SolidToken.SpecFlow.DependencyInjection;
+using static Challenges.API.Tests.Integration.Configuration.Settings;
+
+namespace Challenges.API.Tests.Integration.Support;
+public static class Dependencies
+{
+    private const string APP_SETTINGS_FILE = "appsettings.json";
+
+    [ScenarioDependencies]
+    public static IServiceCollection CreateServices()
+    {
+        var services = new ServiceCollection();
+
+        IConfiguration config = new ConfigurationBuilder()
+            .AddJsonFile(Path.Combine(Directory.GetCurrentDirectory(), APP_SETTINGS_FILE), optional: true, reloadOnChange: true)
```

Comment:
It's not necessary to reload the settings on change. But the file is required, isn't it?
```suggestion
            .AddJsonFile(Path.Combine(Directory.GetCurrentDirectory(), APP_SETTINGS_FILE), optional: false, reloadOnChange: false)
```